Formula/yadm.rb

```diff
@@ -1,21 +1,44 @@
 class Yadm < Formula
   desc "Yet Another Dotfiles Manager"
-  homepage "https://thelocehiliosan.github.io/yadm/"
-  url "https://github.com/TheLocehiliosan/yadm/archive/1.12.0.tar.gz"
-  sha256 "c3d612d01e2027d5f457e0f7d120bc67251b716c373d99fe70638bd86edf107f"
+  homepage "https://yadm.io/"
+  url "https://github.com/TheLocehiliosan/yadm/archive/2.0.0.tar.gz"
+  sha256 "6359debdd9a6154709d084f478c000e572b3d8d50c2abe6525534899a5c2eb16"
 
   bottle :unneeded
 
   def install
+    post_install_message = any_version_installed?
     bin.install "yadm"
     man1.install "yadm.1"
     bash_completion.install "completion/yadm.bash_completion"
     zsh_completion.install  "completion/yadm.zsh_completion" => "_yadm"
+    doc.install "CHANGES"
```

Comment:
These will install automatically, and should not go to `doc`

Reply:
When I remove this, these files are not installed automatically. Can you explain in more detail?

Most notably, I'd like to include the "contrib" directory, which is the place that user contributed hooks are available. It's pretty standard to include contributed supplemental software for users.

Reply:
@fxcoudert - sorry, meant to tag you in the comment above. Can you explain more detail about how those files should install automatically? Thanks.

Reply:
We have a mechanism in place that automatically installs files with certain names: https://github.com/Homebrew/brew/blob/master/Library/Homebrew/metafiles.rb . In your list, `CHANGES` and `LICENSE` should be installed automatically.

Regarding `contrib`, if `make install` does not install it it means upstream does not consider it part of the “normal” installation. You should ask them to include `contrib` in `make install`.

Reply:
@fxcoudert - Thanks for pointing out this. I was simply doing `brew list` (which doesn't show all of the files installed). I can see with `brew list -v` that the `CHANGES`, `LICENSE` are included. 👍

Regarding `contrib`, I'll have it included within `make install` and update once that is available.

I've updated the PR. Please re-review. Thanks.